src/components/AnonymousChat/BaseChannelItem.tsx

```diff
@@ -81,7 +69,7 @@
             unreadCount={unreadCount}
             isMe={isMe}
             hasFollowButton={hasFollowButton}
-            isFollowing={isFollowing}
+            isFollowing={showFollowingButton}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The property `isFollowing` has been replaced with `showFollowingButton`. Ensure that this change is reflected in all places where `BaseChannelItem` is used.
```diff
- isFollowing={isFollowing}
+ isFollowing={showFollowingButton}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->